code/game/objects/items/reagent_containers/reagent_container.dm

```diff
@@ -30,41 +29,29 @@
 
 /obj/item/reagent_containers/attack_hand_alternate(mob/living/user)
 	. = ..()
+	change_transfer_amount(user)
+
+/obj/item/reagent_containers/attack_self_alternate(mob/living/user)
+	if(!change_transfer_amount(user))
+		return ..()
+	SEND_SIGNAL(src, COMSIG_ITEM_ATTACK_SELF_ALTERNATE, user)
+	add_fingerprint(user, "attack_self_alternate")
```

Comment:
Whats going on here?
Same proc defined twice, signal being sent outside of it's correct position (see it previously just being . = ..() i dont see an immediate reason this is this way)

Reply:
If if . = ..()
Then the melee attack chain causes it to run interact() which opens the main UI.
I want it to just open the right-click UI (The TGUI-Input list)
I'm sending the signal and fingerprinting since I want to retain those procs, but excluding the interact() entirely.